src/rdkafka.h

```diff
@@ -4853,11 +4905,22 @@ struct rd_kafka_group_list {
         int group_cnt;                      /**< Group count */
 };
 
+/**
+ * @brief Options for rd_kafka_list_consumer_groups
```

Comment:
These are named rd_kafka_ListConsumerGroups() and .. _DescribeConsumerGroups(), right?

Reply:
These are from the old implementation. I intended to remove them as last thing, going to do it today.